### ISSUE TITLE
app.configure removed in express 4.0

### ISSUE THREAD

**Issue by Aides359** [angular-translate/angular-translate#1622]
### Subject of the issue

As of express 4.0 `app.configure` does not exist anymore (see https://github.com/expressjs/express/issues/936). Thus the demo server does not work.
### Your environment
- version of angular-translate: 2.12.1
### Steps to reproduce

run `grunt server`
### Expected behaviour

The server starts.
### Actual behaviour

```
TypeError: app.configure is not a function                                                                                                                                        
    at Object.<anonymous> (C:\GitStuff\angular-translate\server.js:8:5)                                                                                                           
    at Module._compile (module.js:409:26)                                                                                                                                         
    at Object.Module._extensions..js (module.js:416:10)                                                                                                                           
    at Module.load (module.js:343:32)                                                                                                                                             
    at Function.Module._load (module.js:300:12)                                                                                                                                   
    at Function.Module.runMain (module.js:441:10)                                                                                                                                 
    at startup (node.js:139:18)                                                                                                                                                   
    at node.js:968:3 
```

**_PR in progress_**

**Comment by Aides359** [comment]
Is the server even in use anymore? It looks quite neglected.

**Comment by knalli** [comment]
Well, I don't think so. A simple `serv ./demo` (or any other tool like this) will work also. I think the server is actually not required anymore.

The demo use-case is somehow nice because it shows usage of XHR / loaders.

**Comment by knalli** [comment]
I'll rethink about this, but perhaps we can ditch it completely.

**Comment by knalli** [comment]
Solved in 0e197ce
